be2-scala/src/main/scala/ch/epfl/pop/model/network/method/message/data/DataRegistryModule.scala

```diff
@@ -0,0 +1,31 @@
+package ch.epfl.pop.model.network.method.message.data
+import ch.epfl.pop.model.network.method.message.data.ObjectType._
+import ch.epfl.pop.model.network.method.message.data.ActionType._
+import ch.epfl.pop.model.network.method.message.data.lao.CreateLao
+import ch.epfl.pop.model.network.method.message.data.lao.StateLao
+import ch.epfl.pop.model.network.method.message.data.lao.UpdateLao
+import ch.epfl.pop.model.network.method.message.data.rollCall.CreateRollCall
+import ch.epfl.pop.model.network.method.message.data.rollCall.OpenRollCall
+import ch.epfl.pop.model.network.method.message.data.rollCall.ReopenRollCall
```

Comment:
just to be more concise, you can group the imports

Reply:
agreed, especially as there'll be a ***ton*** of them